### ISSUE TITLE
Load deletes {| ... |}

### ISSUE THREAD

**Issue by andrejbauer** [markokoleznik/agda-writer#11]
On Load in the following code `sup ?` disappears:

```
 module Foo where

 data bool : Set where
   false : bool
   true : bool

 or : bool -> bool -> bool
 or false y = y
 or true _ = true

 data N : Set where
   zero : N
   succ : N -> N

 data Ordinal : Set where
   zer : Ordinal
   suc : Ordinal -> Ordinal
   sup : (N -> Ordinal) -> Ordinal

 sum : Ordinal -> Ordinal -> Ordinal
 sum zer y = y
 sum (suc x) y = suc (sum x y)
 sum (sup xs) y = {! sup ? !}
```

**Comment by andrejbauer** [comment]
Probably related to #12

**Comment by markokoleznik** [comment]
Fixed with #8.
I have no idea why I deleted those goals by hand, I should know that Agda know better when goals are returned :)
